CHANGELOG.md

```diff
@@ -90,39 +90,37 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 	
 - Added new primitive drawing functions to `PrimitiveMan`:  
 	```
-	//
-	PrimitiveMan:DrawArcPrimitive(Vector pos, float startAngle, float endAngle, short radius, unsigned char color)
-	PrimitiveMan:DrawArcPrimitive(Vector pos, float startAngle, float endAngle, short radius, unsigned char color, short thickness)
-	PrimitiveMan:DrawArcPrimitive(short player, Vector pos, float startAngle, float endAngle, short radius, unsigned char color)
-	PrimitiveMan:DrawArcPrimitive(short player, Vector pos, float startAngle, float endAngle, short radius, unsigned char color, short thickness)
-
-	//
-	PrimitiveMan:DrawSplinePrimitive(Vector start, Vector guideA, Vector guideB, Vector end, unsigned char color)
-	PrimitiveMan:DrawSplinePrimitive(short player, Vector start, Vector guideA, Vector guideB, Vector end, unsigned char color)
-
-	//
-	PrimitiveMan:DrawRoundBoxPrimitive(Vector start, Vector end, float cornerRadius, unsigned char color)
-	PrimitiveMan:DrawRoundBoxPrimitive(short player, Vector start, Vector end, float cornerRadius, unsigned char color)
-
-	//
-	PrimitiveMan:DrawRoundBoxFillPrimitive(Vector start, Vector end, float cornerRadius, unsigned char color)
-	PrimitiveMan:DrawRoundBoxFillPrimitive(short player, Vector start, Vector end, float cornerRadius, unsigned char color)
-
-	//
-	PrimitiveMan:DrawTrianglePrimitive(Vector pointA, Vector pointB, Vector pointC, unsigned char color)
-	PrimitiveMan:DrawTrianglePrimitive(short player, Vector pointA, Vector pointB, Vector pointC, unsigned char color)
-
-	//
-	PrimitiveMan:DrawTriangleFillPrimitive(Vector pointA, Vector pointB, Vector pointC, unsigned char color)
-	PrimitiveMan:DrawTriangleFillPrimitive(short player, Vector pointA, Vector pointB, Vector pointC, unsigned char color)
-
-	//
-	PrimitiveMan:DrawEllipsePrimitive(Vector pos, short horizRadius, short vertRadius, unsigned char color)
-	PrimitiveMan:DrawEllipsePrimitive(short player, Vector pos, short horizRadius, short vertRadius, unsigned char color)
-
-	//
-	PrimitiveMan:DrawEllipseFillPrimitive(Vector pos, short horizRadius, short vertRadius, unsigned char color)
-	PrimitiveMan:DrawEllipseFillPrimitive(short player, Vector pos, short horizRadius, short vertRadius, unsigned char color)
+	-- Arc
+	PrimitiveMan:DrawArcPrimitive(Vector pos, startAngle, endAngle, radius, color)
+	PrimitiveMan:DrawArcPrimitive(player, Vector pos, startAngle, endAngle, radius, color)
+	
+	PrimitiveMan:DrawArcPrimitive(Vector pos, startAngle, endAngle, radius, color, thickness)
+	PrimitiveMan:DrawArcPrimitive(player, Vector pos, startAngle, endAngle, radius, color, thickness)
+
+	-- Spline (Bézier Curve)
+	PrimitiveMan:DrawSplinePrimitive(Vector start, Vector guideA, Vector guideB, Vector end, color)
+	PrimitiveMan:DrawSplinePrimitive(player, Vector start, Vector guideA, Vector guideB, Vector end, color)
+
+	-- Box with round corners (Squircle)
```

Comment:
As mention in the relevant primitve drawing methods, I'd go with naming it Rounded corners, though it's less important here. Also, I both love and hate the word `Squircle`

Reply:
Yeah it's a silly word, but it's a real one. :man_shrugging: 
I guess it's technically incorrect to call it that way tho, so i'll just go with rounded box.